src/components/filter-row/filter-row.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classnames from 'classnames';
+import VisibleIcon from '../icons/visible';
```

Comment:
the filter row doesn't have the eye icon, so why are we calling it here?

Reply:
hey @rashidakanchwala I've updated it to IndicatorIcon instead as its for the FilterRow specifically now